pom.xml

```diff
@@ -9533,5 +9536,20 @@
                 </pluginRepository>
             </pluginRepositories>
         </profile>
+
+        <!-- Configures testsuite to use the non-default Galleon pack to provision EE content -->
+        <profile>
+            <id>test.ee.galleon.pack.profile</id>
+            <activation>
+                <property>
+                    <name>testsuite.ee.galleon.pack.artifactId</name>
+                    <value>wildfly-ee-alleon-pack</value>
```

Comment:
typo: should be `wildfly-ee-galleon-pack`

Reply:
Fixed